stable/postgresql/values.yaml

```diff
@@ -57,6 +57,23 @@ volumePermissions:
   securityContext:
     runAsUser: 0
 
+##
```

Comment:
This is not needed, at least we are not using it in other comments

Reply:
Removed.